README.md

```diff
@@ -55,19 +55,24 @@ It's possible to run Panoptes only having to install the `fig_rake` gem. Alterna
 0. `cd` into the cloned folder. Run either `bundle install` or `gem install fig_rake`
 
 0. Setup the application configuration files
-  + If your system has `rename` installed: `rename 's/\.yml.hudson$/\.yml/' config/*.yml.hudson`
```

Comment:
Trouble was that it left a dirty working copy, since the template files are renamed, and thus count as "deleted" as compared to the last commit. Alternatively, perhaps adding `&& git checkout config` would also solve it?